lte/gateway/c/session_manager/SessionStateEnforcer.cpp

```diff
@@ -576,12 +576,11 @@ void SessionStateEnforcer::prepare_response_to_access(
   /* AMBR value need to compared from AMF and PCF, then fill the required
    * values and sent to AMF.
    */
-  // For now its default QOS, AMBR has default values
-  rsp->mutable_downlink_unit_type()->set_ambr_unit_type(
-      magma::AmbrUnit::Kbps_64);
-  rsp->mutable_uplink_unit_type()->set_ambr_unit_type(magma::AmbrUnit::Kbps_64);
-  rsp->set_downlink_units(DEFAULT_AMBR_UNITS);
-  rsp->set_uplink_units(DEFAULT_AMBR_UNITS);
+  // For now its default QOS, AMBR is 1 Gbps downlink
+  rsp->mutable_session_ambr()->set_br_unit(AggregatedMaximumBitrate::KBPS);
+  rsp->mutable_session_ambr()->set_max_bandwidth_ul(DEFAULT_AMBR_UNITS);
+  rsp->mutable_session_ambr()->set_max_bandwidth_dl(DEFAULT_AMBR_UNITS);
+

```

Comment:
can you please point me to any unit test that tests this function `prepare_response_to_access`

Reply:
prepare_response_to_access : No direct UT for it.  Extended coverage for m5g_move_to_active_state() which calls the function internally.